packages/git/src/common/git.ts

```diff
@@ -562,4 +637,39 @@ export namespace GitUtils {
         return false;
     }
 
+    /**
+     * Maps the raw status text from Git to a Git file status enumeration.
+     */
+    export function mapStatus(rawStatus: string): GitFileStatus {
```

Comment:
is that used by anything ?

Reply:
No, not yet. Just like the `exec` function, but we would like to provide flexibility for future extension developers with the BETA release.

Reply:
It is defined to be used by potential extension developers but it should be tested at least.
@kittaakos could you add one?